hie.yaml

```diff
@@ -1,25 +0,0 @@
-cradle:
```

Comment:
I do use HLS but have no opinion if it should be part of the repo.

Reply:
`implicit-hie` is called in HLS making these files deprecated unless you want to only have specific targets in scope.

Reply:
But also, add it to your .gitignore and use it as you please :)